commoneventconsumer/src/main/scala/com/gu/notifications/events/model/EventAggregation.scala

```diff
@@ -79,6 +120,12 @@ object EventAggregation {
     (value: JsValue) => value.validate[Map[String, Int]].map(_.map { case (k, v) => (LocalDateTime.parse(k), v) }),
     (dateTimeMap: Map[LocalDateTime, Int]) => Json.toJsObject(dateTimeMap.map { case (k, v) => (k.toString, v) })
   )
-  implicit val jf = Json.format[EventAggregation]
+
+  implicit val jreads = Json.reads[EventAggregation]
+  implicit val jwrites: Writes[EventAggregation] =
+    Json
+      .writes[EventAggregation]
+      .addField("timingPercentiles", _.timingPercentiles)
+      .removeField("timing")
```

Comment:
I made the decision to hide the timing map. It is super long and doesn't provide much information as is.

Reply:
No I agree! we just need to double check if that has an impact on the event consumer that stores the data in dynamo

Reply:
I double-checked, it's okay to hide it like that :+1: